olive/passes/onnx/extract_adapters.py

```diff
@@ -350,3 +373,23 @@ def _create_empty_initializer(cls, dag: OnnxDAG, weights: Dict[str, "NDArray"],
         new_initializer = cls._copy_initializer(old_proto, new_name)
         # add the new initializer to the graph
         dag.add_initializer(new_initializer, dag.get_io(old_name).graph_idx)
+
+    @classmethod
+    def _make_dynamic_optional(cls, dag: OnnxDAG, weights: Dict[str, "NDArray"], name: str, config: Dict[str, Any]):
+        """Make the input dynamic and optional."""
+        if "quant" in name:
+            # dynamic shape and optional inputs not supported for quantized modules yet
```

Comment:
nit pick: add warning.

Reply:
Will do the config check first and log a warning here for quant models.

Reply:
Done!